compiler-plugin/src/main/java/io/ballerina/stdlib/data/jsondata/compiler/Constants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.data.jsondata.compiler;
+
+/**
+ * Constants for Jsondata's compiler plugin.
```

Comment:
```suggestion
 * Constants for Jsondata's compiler plugin.
 * @since 0.1.0
```

Apply other where applicable